fontspec-doc-featset.tex

```diff
@@ -608,6 +608,66 @@ \subsection{Letter spacing}
 In particular, small amounts of letter spacing can be very useful, when setting small caps or all caps titles.
 Also see the OpenType \opt{Uppercase} option of the \feat{Letters} feature (\vref{sec:letters}).
 
+
+\section{Variable fonts} \label{sec:variable}
```

Comment:
Not sure if this is the best place for this, but since it's not really OpenType specific due to the MM stuff I thought that a generic place is best.

Reply:
It's okay for now. The situation would anything non-OpenType fontspec is rather weird and uncertain